src/GitReleaseManager.Cli/GitReleaseManager.Cli.csproj

```
@@ -3,7 +3,7 @@
         <LangVersion>8.0</LangVersion>
         <OutputType>Exe</OutputType>
         <AssemblyName>GitReleaseManager</AssemblyName>
-        <TargetFrameworks>net48;net6.0;net7.0</TargetFrameworks>
+        <TargetFrameworks>net6.0</TargetFrameworks>
```

```suggestion
        <TargetFrameworks>net6.0</TargetFrameworks>
        <PublishSingleFile>true</PublishSingleFile>
        <SelfContained>true</SelfContained>
        <RuntimeIdentifier>win-x64</RuntimeIdentifier>
```

I "think" we should add something like the following.  The will allow the creation of a single self contained exe, which will we able to run straight away, without the need to install a specific runtime.

@AdmiringWorm did I miss anything here?

---

Actually...

Should we be using `win-x86`, rather than `win-x64`?  Previously, I think we would have been compiling for AnyCPU, maybe?

---

The suggestion for adding these properties comes from this help page on the Microsoft Documentation: https://learn.microsoft.com/en-us/dotnet/core/deploying/single-file/overview?tabs=cli